package.json

```diff
@@ -4,12 +4,21 @@
   "description": "Practice exercises to learn the syntax for testing javascript code.",
   "main": "index.js",
   "scripts": {
-    "test": "jest --watch"
+    "test": "jest --watch --verbose"
```

Comment:
It's always good to make your tests verbose so you can see which ones passed and which ones failed 👍